content/en/docs/collector/deployment/gateway.md

```diff
@@ -212,6 +212,47 @@ Cons:
 - Added latency in case of cascaded collectors
 - Higher overall resource usage (costs)
 
+## Additional Considerations When Deploying Multiple Collectors
+
+### The Single-Writer Principle
+
+The Single-Writer Principle is a design principle that ensures a single logical writer for a particular resource.
+Concurrent access from multiple applications that modify or report on the same data can lead to data
+loss or, at least, degraded data quality. In gateway collector deployments, applying this principle guards against sending
+inconsistent data to the backend.
+
+#### Context in OTLP
+
+All metric data streams within OTLP must have a [single writer](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer)
+This is because gateway collector deployments can involve multiple collectors in the same system.
+It is possible in this case for instances to receive and process data from the same resources,
+which is a violation of the Single-Writer principle. A potential consequence of this may be that
```

Comment:
I'm not sure I agree with this. Here's what the link above says:

> All metric data streams within OTLP MUST have one logical writer. This means, conceptually, that any Timeseries created from the Protocol MUST have one originating source of truth. In practical terms, this implies the following:
>
> - All metric data streams produced by OTel SDKs SHOULD have globally unique identity at any given point in time. [Metric identity is defined above.](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#opentelemetry-protocol-data-model-producer-recommendations)
> - Aggregations of metric streams MUST only be written from a single logical source at any given point time. Note: This implies aggregated metric streams must reach one destination.

Multiple instances of the collector can certainly receive and process different OTLP requests for the same resource without problems. 

In the context of the collector, the single-writer principle is relevant for receivers that create metrics on their own, such as scraping receivers or things like the host metrics receiver. On that case, there should really be only one "host metrics receiver" instance per host.

Reply:
Ok, makes sense. This is a much more particular problem than my framing would suggest. 

In hindsight, 

> It is possible in this case for instances to receive and process data from the same resources,
which is a violation of the Single-Writer principle. A potential consequence of this may be that

Is not really correct. That's far too general, especially considering the case you've highlighted.

Thanks for the feedback, I will get to revising this.